core/text/text_and_dictation.py

```diff
@@ -120,14 +120,28 @@ def word(m) -> str:
         )
 
 
-@mod.capture(rule="({user.vocabulary} | <phrase>)+")
+@mod.capture(rule="({user.vocabulary} | <user.prose_contact> | <phrase>)+")
```

Comment:
We could also add this to <user.word> above -- just bear in mind that it'll take multiple words to speak it, and some result in multiple words.